game/menu/pausemenu/pausemenu.gd

```diff
@@ -0,0 +1,70 @@
+extends CanvasLayer
+
+signal pause_menu_opened
+signal pause_menu_closed
+
+onready var _main := $Margin
+onready var _resume_btn := $Margin/VBox/resumebtn
+onready var _reset_btn := $Margin/VBox/restartbtn
+onready var _quit_game_btn := $Margin/VBox/quitgamebtn
+onready var _quit_desc_btn := $Margin/VBox/quittodesctopbtn
```

Comment:
I like to use PascalCase for node names.
And when i have to use them in function names => snake_case :P